From 4d96bee2206ac12fcd850e81408d9df2e6078f03 Mon Sep 17 00:00:00 2001
From: Han Gao <gaohan@iscas.ac.cn>
Date: Sun, 10 May 2026 20:51:37 +0800
Subject: [PATCH 81/99] RUYI: riscv: dts: milkv-megrez: enable SATA

Enable the on-board SATA M.2 socket on the Milk-V Megrez board.

Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts b/arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts
index 5f4a8e79aac6..54b520f6d64a 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-milkv-megrez.dts
@@ -135,6 +135,14 @@ &pcie {
 	status = "okay";
 };
 
+&sata {
+	status = "okay";
+};
+
+&sata_phy {
+	status = "okay";
+};
+
 &sdhci_emmc {
 	status = "okay";
 	/delete-property/ mmc-hs400-1_8v;
-- 
2.53.0

